rust/src/lib/ovsdb/db.rs

```diff
@@ -421,3 +473,99 @@ impl OvsDbUpdate {
         Value::Object(ret)
     }
 }
+
+pub fn string_to_ovn_bridge_mappings(
+    mappings_string: String,
+) -> Result<Option<Vec<OvnBridgeMapping>>, NmstateError> {
+    let mut mappings: Vec<OvnBridgeMapping> = Vec::new();
+    for mapping_str in mappings_string.split(',') {
+        match OvnBridgeMapping::from_str(mapping_str) {
+            Ok(mapping) => mappings.push(mapping),
+            Err(e) => {
+                return Err(NmstateError::new(InvalidArgument, e.to_string()))
+            }
+        }
+    }
+    if mappings.is_empty() {
+        Ok(None)
+    } else {
+        Ok(Some(mappings))
+    }
+}
+
+#[cfg(test)]
+mod tests {
```

Comment:
Please move test code to folder `rust/src/lib/unit_tests`

Reply:
I had it there, but I honestly couldn't get this to pass the linters because of the rust modules. I'll need some help with that.

Reply:
Done.